azerothcore/azerothcore-wotlk
# fix(DB/Quests) Report to Helgrum prerequisite

## Changes Proposed:
- Adds [Neeka Bloodscar](https://tbc.wowhead.com/quest=1418/neeka-bloodscar) as a required previous quest for [Report to Helgrum](https://tbc.wowhead.com/quest=1420/report-to-helgrum)

## Issues Addressed:
- closes https://github.com/azerothcore/azerothcore-wotlk/issues/6720
- closes https://github.com/chromiecraft/chromiecraft/issues/1082

## SOURCE:
https://tbc.wowhead.com/quest=1420/report-to-helgrum

## Tests Performed:
- sql script executes without errors 
- tested ingame


## How to Test the Changes:
Make sure you didn't complete the quests with the ids 1418 and 1420. (If you are not sure, run `.q remove 1418` and `.q remove 1420`.

1. `.go c 6874` 
2. Check if Neeka Bloodscar has the quest [Report to Helgrum](https://tbc.wowhead.com/quest=1420/report-to-helgrum) available
3. `.q add 1418`
4. `.q complete 1418`
5. Turn in the Quest at Neeka Blooscar
6. Now Neeka Bloodscar should provide [Report to Helgrum](https://tbc.wowhead.com/quest=1420/report-to-helgrum) 

## Known Issues and TODO List:
N/A


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
